tests/testAPI.sh

```diff
@@ -21,6 +21,29 @@ KRUIZE_URL="http://localhost"
 KRUIZE_PORT="31313"
 AUTH_TOKEN=""
 
+function usage() {
+	echo "Usage: $0 [-u kruize_url] [-t kruize_port] [-a auth_token]"
+	exit -1
+}
+
+# Iterate through the commandline options
+while getopts u:t:a: gopts
```

Comment:
Nit: Can you please arrange this alphabetically, ie, a, t and u.

Reply:
Arranged alphabetically